api/db-cdr/changelog-local/db.changelog-5.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="brianfreeman" id="changelog-5" runAlways="true">
+
+    <validCheckSum>ANY</validCheckSum>
+
+    <loadData tableName="criteria" file="csv/icd10_criteria-local.csv" encoding="UTF-8" quotchar='"'>
```

Comment:
While you're at it, should we make a smaller version of ICD 9 so local builds go faster?

Reply:
icd9 file is about the same size as the subset icd10 file. It's pretty fast currently(2.6s).

changelog-1::brianfreeman ran successfully in 14ms
changelog-2::brianfreeman: Table criteria created
changelog-2::brianfreeman: Index idx_type_parent_id created
changelog-2::brianfreeman: ChangeSet changelog/db.changelog-2.xml::changelog-2::brianfreeman ran successfully in 27ms
changelog-3::brianfreeman: Data loaded from csv/icd9_criteria.csv into criteria
changelog-3::brianfreeman: ChangeSet changelog/db.changelog-3.xml::changelog-3::brianfreeman ran successfully in 2614ms
changelog-4::brianfreeman: Data loaded from csv/demo_criteria.csv into criteria
changelog-4::brianfreeman: ChangeSet changelog/db.changelog-4.xml::changelog-4::brianfreeman ran successfully in 7ms
changelog-5::brianfreeman: Data loaded from csv/icd10_criteria-local.csv into criteria
changelog-5::brianfreeman: ChangeSet changelog-local/db.changelog-5.xml::changelog-5::brianfreeman ran successfully in 1848ms

Reply:
And I will add comments to README

Reply:
I updated the the README at the very bottom under API Server Database Updates.